js/latoken.js

```diff
@@ -242,7 +236,7 @@ module.exports = class latoken extends Exchange {
         //         }
         //     ]
         //
-        if (this.safeValue (this.options, 'adjustForTimeDifference', true)) {
```

Comment:
this change is breaking latoken

Reply:
I don't think we should remove the guards from `this.options` that seems like a regression to me

Reply:
it's breaking anyway, changing this back won't fix it